WORKSPACE

```diff
@@ -221,3 +221,8 @@ http_archive(
         "https://github.com/bazelbuild/bazel-toolchains/archive/cdea5b8675914d0a354d89f108de5d28e54e0edc.tar.gz",
     ],
 )
+
+
+register_toolchains(
```

Comment:
add doc that this registers the default docker toolchain that expects docker on path